addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua

```diff
@@ -41,10 +41,10 @@ if CLIENT then
   language.Add( "tool."..gsToolModeOP..".analog_con"     , "Analog input" )
   language.Add( "tool."..gsToolModeOP..".config"         , "Click this button to open the joystick configuration. You can also right click on the world" )
   language.Add( "tool."..gsToolModeOP..".config_con"     , "Joystick Configuration" )
-  language.Add( "undone_"..gsToolModeOP, "Undone Wire Joystick Multi" )
-  language.Add( "sboxlimit_"..gsToolLimits, "You've hit the Joystick Multi limit!" )
-  language.Add( "cleanup_" .. gsToolLimits, "Wire Joystick Multi chips " )
-  language.Add( "cleaned_" .. gsToolLimits, "Cleaned up all Joystick Multi chips" )
+  language.Add( "undone_"..gsToolModeOP                  , "Undone Wire Joystick Multi !" )
```

Comment:
Why is there a space at end before the exclamation mark?

Reply:
Reverted in d1a97f191828337d3eb16f73ce41a6784df705a1